homework/java/BuchynskiM_JHL1T2/L3/Task1_DogDemo.java

```diff
@@ -0,0 +1,62 @@
+package Lecture3_Task1;
+
+
+public class DogDemo {
+    public static void main(String[] args) {
+
+        Dog dog = new Dog(1, "white", "Bobik");
+
+        dog.voice();
+        dog.eat();
+        dog.sleep();
+
+    }
+
+    static class Dog{
+        private int age;
+        private String color;
+        private String name;
+
+        public Dog(int age, String color, String name) {
+            this.age = age;
+            this.color = color;
+            this.name = name;
+        }
+
+        public int getAge() {
+            return age;
+        }
+
+        public void setAge(int age) {
+            this.age = age;
+        }
+
+        public String getColor() {
+            return color;
+        }
+
+        public void setColor(String color) {
+            this.color = color;
+        }
+
+        public String getName() {
+            return name;
+        }
+
+        public void setName(String name) {
+            this.name = name;
+        }
+
+        public void voice(){
+            System.out.println(name+" is barking");
```

Comment:
please input spaces around concotenations for more readable code

Reply:
ok. Thank you. The changes has been done